assignment/assignment.hs

```diff
@@ -31,7 +32,10 @@ factorial n = 0 -- replace me
 -- You may also need list concatenation, `++`.
 
 toDigits :: Int -> [Char]
-toDigits num = "replace me"
+toDigits n = if(n < 10) then (intToDigit n):[] else toDigits tr ++ (intToDigit r):[]
+  where (tr, r) = divMod n 10 
+  --tr is truncated result, r is remainder. I originally had 'tres' and 'rem' but it threw an error 
+  --also i had to make the chars into [char] by using :[]. This feels hacky so I was wondering if there's a nicer way
 
```

Comment:
You could just put it in brackers: `[ intToDigit r]`. But your way is also OK.

I tend to like laying by code out on multiple lines, just because my tired old brain finds it easier to read if each line does just a few things:


~~~ hs
toDigits :: Int -> [Char]
toDigits n 
    | n < 10    = [ intToDigit n ]
    | otherwise = toDigits rest ++ toDigits lowOrder 
    where
        ( rest, lowOrder ) = divMod n 10
~~~